src/plugins/tabs/components/tabs.vue

```diff
@@ -224,15 +226,21 @@ export default {
             this.setCurrentTab = true;
         },
         removeItem(identifier) {
-            let pos = this.tabsList.findIndex(tab =>
-                tab.domainObject.identifier.namespace === identifier.namespace && tab.domainObject.identifier.keyString === identifier.keyString
-            );
+            let keyStringToBeRemoved = this.openmct.objects.makeKeyString(identifier);
+
+            let pos = this.tabsList.findIndex(tab => {
+                return tab.keyString === keyStringToBeRemoved;
+            });
+
             let tabToBeRemoved = this.tabsList[pos];
 
             tabToBeRemoved.statusUnsubscribe();
 
             this.tabsList.splice(pos, 1);
 
+            this.loadedTabs[keyStringToBeRemoved] = undefined;
+            delete this.loadedTabs[keyStringToBeRemoved];
```

Comment:
is it reactive? to be safe can use this `this.$delete(target, propertyName/index )` https://vuejs.org/v2/api/#vm-delete

Reply:
We don't the view to react to this delete change, so there is no need to use the vue $delete method.